source/public_collections_advanced_search.md

```diff
@@ -0,0 +1,104 @@
+# Public Collections Advanced Search
+
+**_Please note that the search capability is a work in progress, and we plan to
+add a more user-friendly syntax in future releases._**
+
+### The Default Search Behaviour
+
+When you enter a term into the search bar without specifying a field, your
+query will return results where that term, regardless of capitalization, can be
+found in:
+- The name of the survey, collection, question, block, or template;
+- The owner's username;
+- The description;
+- The summary, which contains all question labels; [and maybe other stuff? I
+  don't remember]
```

Comment:
\*giggle\* replace my stuff between the brackets with a more complete description of summary, or just trash it if time doesn't permit :smile:

Reply:
😂 @jnm I was clearly paying attention while doing this